infra/upload_images/container_ocr/handler/ocr_processor.py

```
@@ -199,6 +204,484 @@ def _process_refinement_job(
+        page, lek = self.dynamo.list_receipt_word_labels_for_receipt(
+            image_id=ocr_job.image_id, receipt_id=ocr_job.receipt_id
+        )
+        labels.extend(page or [])
+        while lek:
+            page, lek = self.dynamo.list_receipt_word_labels_for_receipt(
+                image_id=ocr_job.image_id,
+                receipt_id=ocr_job.receipt_id,
+                last_evaluated_key=lek,
+            )
+            labels.extend(page or [])
+        labeled_keys = {(lbl.line_id, lbl.word_id) for lbl in labels}
```

Ruff (0.15.1)

[error] Local variable `labeled_keys` is assigned to but never used

Remove assignment to unused variable `labeled_keys`

(F841)